yun/libraries/Parse/examples/test/Temperature.json

```diff
@@ -0,0 +1,108 @@
+{ "results": [
+	{
+        "ACL": {
+            "*": {
+                "read": true,
+                "write": true
+            },
+            "bgneIdDXc0": {
+                "read": true,
+                "write": true
+            }
+        },
+        "createdAt": "2015-03-23T23:28:35.710Z",
+        "leverDown": true,
+        "objectId": "I4S0FgSeqT",
+        "s": "1",
```

Comment:
Could we delete the s key from the JSON, or rename it stringField?  It's potentially confusing because its not descriptive and has strings that look like ints.